mcc/ttl/main.go

```diff
@@ -255,7 +257,7 @@ func main() {
 	if len(list2) > 0 {
 		list = list2
 	}
-	changeResponse, err := UpsertResourceRecordSetTTL(list, *ttl, zoneID, svc)
+	changeResponse, err := UpsertResourceRecordSetTTL(list, ttl, zoneID, svc)
```

Comment:
Being `ttl` a global variable, do we still need to pass it?
Maybe doing `svc` a global variable could also help... but I'm guessing some of the following PRs are doing other things related to these.